src/tts_text/phoneme_covering_set.py

```diff
@@ -0,0 +1,196 @@
+"""Module for creating a phoneme covering set."""
+import json
+import re
+import logging
+from collections import Counter
+from pathlib import Path
+from tqdm import tqdm
```

Comment:
Enables running in notebooks.

```suggestion
from tqdm.auto import tqdm
```